### ISSUE TITLE
for BHL, track meta.xml as well as the extracted text

### ISSUE THREAD

**Issue by jhpoelen** [bio-guoda/preston#296]
fyi @myrmoteras 

Currently, only the bhl item index and their associated ocr text are tracked.

suggest to include meta.xml for each item also, so that more context for each item can be retrieved.

**Comment by jhpoelen** [comment]
Currently tracking 

https://ia600701.us.archive.org/0/items/mobot31753000022803/mobot31753000022803_meta.xml

or

```xml
<?xml version="1.0" encoding="UTF-8"?>
<metadata>
  <identifier>mobot31753000022803</identifier>
  <call-number>QK98 .R6 1789</call-number>
  <contributor>Missouri Botanical Garden</contributor>
  <curation>[curator]botanicus.org[/curator][date]20111003041428[/date][state]approved[/state]</curation>
  <date>1789</date>
  <language>UND</language>
  <mediatype>texts</mediatype>
  <possible-copyright-status>Permission to digitize granted by rights holder</possible-copyright-status>
  <publisher>Paris : Poincot, Libraire,</publisher>
  <rights>http://www.biodiversitylibrary.org/permissions</rights>
  <sponsor>Missouri Botanical Garden</sponsor>
  <title>Recueil de plantes coloriees, pour servir a l'intelligence des lettres elementaires sur la botanique.</title>
  <tl2author>Rousseau, Jean-Jacques</tl2author>
  <collection>botanicus</collection>
  <collection>biodiversity</collection>
  <creator>Rousseau, Jean-Jacques,</creator>
  <subject>Botany</subject>
  <subject>Pictorial works</subject>
  <publicdate>2011-10-03 09:14:29</publicdate>
  <uploader>botanicus.upload@gmail.com</uploader>
  <addeddate>2011-10-03 09:14:29</addeddate>
  <ppi>400</ppi>
  <imagecount>193</imagecount>
  <foldoutcount>0</foldoutcount>
  <identifier-access>http://www.archive.org/details/mobot31753000022803</identifier-access>
  <identifier-ark>ark:/13960/t7wm28d9s</identifier-ark>
  <repub_state>4</repub_state>
  <backup_location>ia903703_20</backup_location>
  <openlibrary_edition>OL25471139M</openlibrary_edition>
  <openlibrary_work>OL16845908W</openlibrary_work>
  <ocr>tesseract 5.0.0-1-g862e</ocr>
  <ocr_parameters>-l deu+rus+ita+que+fra+kaz+spa+bel+bul+kir+eng+Latin+Cyrillic+Arabic</ocr_parameters>
  <ocr_module_version>0.0.14</ocr_module_version>
  <ocr_autonomous>true</ocr_autonomous>
  <ocr_detected_script>Latin</ocr_detected_script>
  <ocr_detected_script>Cyrillic</ocr_detected_script>
  <ocr_detected_script>Arabic</ocr_detected_script>
  <ocr_detected_script_conf>0.7446</ocr_detected_script_conf>
  <ocr_detected_script_conf>0.1352</ocr_detected_script_conf>
  <ocr_detected_script_conf>0.0495</ocr_detected_script_conf>
  <ocr_detected_lang>fr</ocr_detected_lang>
  <ocr_detected_lang_conf>1.0000</ocr_detected_lang_conf>
  <ocr_invalid_language>UND</ocr_invalid_language>
  <pdf_module_version>0.0.17</pdf_module_version>
</metadata>
```

but could also switch to tracking _marc.xml or _dc.xml instead.

**Comment by jhpoelen** [comment]
available through preston v0.8.6
